test/validation/api/scheduler/scheduler.c

```diff
@@ -629,6 +629,117 @@ static void scheduler_test_full_queues(void)
 	CU_ASSERT(odp_pool_destroy(pool) == 0);
 }
 
+static void scheduler_test_max_queues(odp_schedule_sync_t sync)
```

Comment:
Looking good.

One thing I noticed while reviewing this PR is that `odp_schedule_capability_t.lockfree_queues` and `odp_schedule_capability_t.waitfree_queues` are not tested at all by the validation suite. This can be fixed in a separate PR.

Reply:
I'm planning to do a new PR which creates queues with all param combinations, including these.